extensions/blocks/simple-payments/index.js

```diff
@@ -120,6 +120,16 @@ export const settings = {
 
 	save,
 
+	example: {
+		attributes: {
+			price: 25.0,
+			title: 'Jetpack t-shirt',
+			content:
+				'Take flight in ultimate comfort with ths stylish t-shirt featuring the Jetpack logo.',
```

Comment:
Remember to add `__()`s. This string will be a funky one for translators to see. :-D

Reply:
Will do!

Reply:
Added translations